clouddriver-ecs/src/test/groovy/com/netflix/spinnaker/clouddriver/ecs/security/EcsCredentialsLifeCyclerHandlerSpec.groovy

```diff
@@ -45,15 +45,16 @@ class EcsCredentialsLifeCyclerHandlerSpec extends Specification {
     Set<Class> expectedClasses = [ IamRoleCachingAgent.class, EcsClusterCachingAgent.class, ServiceCachingAgent.class,
                          TaskCachingAgent.class, ContainerInstanceCachingAgent.class, TaskDefinitionCachingAgent.class,
                          TaskHealthCachingAgent.class, EcsCloudMetricAlarmCachingAgent.class, ScalableTargetsCachingAgent.class,
-                         SecretCachingAgent.class, ServiceDiscoveryCachingAgent.class, TargetHealthCachingAgent.class ]
+                         SecretCachingAgent.class, ServiceDiscoveryCachingAgent.class, TargetHealthCachingAgent.class,
+                         ApplicationCachingAgent.class ]
     Set<Class> actualClasses =[]
 
     when:
     handler.credentialsAdded(credOne)
 
     then:
     1 * ecsAccountMapper.addMapEntry({it.getName() == credOne.getName()})
-    ecsProvider.getAgents().size() == 23 // 2 * 12 - 1 ( One IamRoleCachingAgent per account )
+    ecsProvider.getAgents().size() == 24 // 2 * 12 - 1 ( One IamRoleCachingAgent per account )
```

Comment:
now that this is 24, the formula in the comment doesn't make sense (if `2` = # regions, `12` = # of agents per region, , `1` = agents only 1 per account, it's now `(2 * 13) - 2` )? I can't immediately tell where the 2 regions comes from, though

Reply:
The 2 regions is actually defined in TestCredentials.java (region 1 is us-east-1 and 2 is us-west-1): https://github.com/spinnaker/clouddriver/blob/master/clouddriver-ecs/src/test/java/com/netflix/spinnaker/clouddriver/ecs/TestCredential.java#L39-L45